internal/hmyapi/apiv2/types.go

```diff
@@ -528,21 +540,16 @@ func newRPCStakingTransactionFromBlockIndex(b *types.Block, index uint64) *RPCSt
 	return newRPCStakingTransaction(txs[index], b.Hash(), b.NumberU64(), b.Time().Uint64(), index)
 }
 
-// CallArgs represents the arguments for a call.
-type CallArgs struct {
-	From     *common.Address `json:"from"`
-	To       *common.Address `json:"to"`
-	Gas      *hexutil.Uint64 `json:"gas"`
-	GasPrice *hexutil.Big    `json:"gasPrice"`
-	Value    *hexutil.Big    `json:"value"`
-	Data     *hexutil.Bytes  `json:"data"`
-}
-
-// StakingNetworkInfo returns global staking info.
-type StakingNetworkInfo struct {
-	TotalSupply       numeric.Dec `json:"total-supply"`
-	CirculatingSupply numeric.Dec `json:"circulating-supply"`
-	EpochLastBlock    uint64      `json:"epoch-last-block"`
-	TotalStaking      *big.Int    `json:"total-staking"`
-	MedianRawStake    numeric.Dec `json:"median-raw-stake"`
-}
+// getRPCLeader returns either the leader one address (before staking)
+// or the first 20 bytes of the leader bls public key (after staking)
+func getRPCLeader(h *block.Header, isStaking bool) string {
+	if isStaking {
+		return strings.ToLower(h.Coinbase().Hex())
```

Comment:
Let's derive the correct ONE1 address and report that instead of this random hex that people don't recognize.

you can refer to this code for the derivation:

```
for _, member := range committee.Slots {
		if isStaking {
			// After staking the coinbase address will be the address of bls public key
			if utils.GetAddressFromBLSPubKeyBytes(member.BLSPublicKey[:]) == header.Coinbase() {
				if err := member.BLSPublicKey.ToLibBLSPublicKey(committerKey); err != nil {
					return nil, err
				}
				return &bls.PublicKeyWrapper{Object: committerKey, Bytes: member.BLSPublicKey}, nil
			}```